### DISCUSSION TITLE
real-time logs delivery

### DISCUSSION THREAD

**Discussion by AndreiBanaruTakeda** [bottlerocket-os/bottlerocket#4180] [Q&A] [answered]
When it comes to compliance and audits, how are people capturing logs from the Bottlerocket host OS to send them to a remote destination (in real-time)?

I'm interested, for now, in kubelet and auditd logs.

**Comment by arnaldo2792** [comment]
Hey @AndreiBanaruTakeda , thanks for asking! Have you have a chance to look at this blogpost?

https://opensearch.org/blog/bottlerocket-k8s-fluent-bit/

It explains how to collect kubelet logs, but it could be extended to include any system logs you want since the logs are collected from the journal. It uses OpenSearch as the indexing tool but I think you can hook up whatever indexer fluentbit supports.

**Reply by AndreiBanaruTakeda** [reply, marked as answer]
Thanks @arnaldo2792

I did read that blogpost, although I overlooked the journald section, when he gets the kubelet events from it.
In the end I used the splunk-otel-collector, worked out pretty good, pulling from journald.

**Comment by SAAMalusare** [comment]
Hello @arnaldo2792 , I was able to get kernel and kubelet logs but failed to get host, ntp logs (using fluentbit).
Is there any other tech-note available to achieve the same?

thank you,
Suhas
